scripts/exglobal_land_analysis_run.sh

```diff
@@ -0,0 +1,175 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_run.sh
+#
+# This script runs a global land letkfoi analysis with FV3-JEDI.
+# It assumes the runtime directory has already been staged with the appropriate
+# input files and YAML configuration (by the initialize script) before execution.
+#
+################################################################################
+#  Utilities
+export NCP=${NCP:-"/bin/cp"}
+export NMV=${NMV:-"/bin/mv"}
+export NLN=${NLN:-"/bin/ln -sf"}
+CASE=${CASE:-C96}
+RES=$(echo "${CASE}" | cut -c 2-)
+
+#  Directories.
+export gdasapp_dir=${gdasapp_dir:-"${HOMEgfs}/sorc/gdas.cd/"}
+BKGDIR=${DATA}/bkg
+ANLDIR=${DATA}/anl
+
+#  Filenames
+export CREATEENS=${CREATEENS:-"${HOMEgfs}/sorc/gdas.cd/ush/land/letkf_create_ens.py"}
+export IMS_IODA=${IMS_IODA:-"${HOMEgfs}/sorc/gdas.cd/build/bin/imsfv3_scf2ioda.py"}
+export CALCFIMS=${CALCFIMS:-"${HOMEgfs}/sorc/gdas.cd/build/bin/calcfIMS.exe"}
+export ADDJEDIINC=${ADDJEDIINC:-"${HOMEgfs}/sorc/gdas.cd/build/bin/apply_incr.exe"}
+
+#export TPATH="/scratch2/NCEPDEV/land/data/fix/C${RES}/"
+#export TPATH=${FIXgfs:-"${HOMEgfs}/fix/orog/C${RES}"}
+export TPATH="${HOMEgfs}/fix/orog/C${RES}"
+export TSTUB="C${RES}_oro_data"
+export GFSv17=${GFSv17:-"NO"}
```

Comment:
Not a fan of this variable name at all. Can you use something more meaningful as to what the setting actually is?

Reply:
Made changes to all three variable names.